java/vector/src/main/java/org/apache/arrow/vector/ZeroVector.java

```diff
@@ -39,7 +41,7 @@
 public class ZeroVector implements FieldVector {
   public final static ZeroVector INSTANCE = new ZeroVector();
 
-  private final String name = "[DEFAULT]";
+  public static final String NAME = "[DEFAULT]";
```

Comment:
If this name is just used for the conversion, can we move it there?

Reply:
done